### ISSUE TITLE
Version 535: All text on whole page flickering rapidly

### ISSUE THREAD

**Issue by bhrgunatha** [YePpHa/YouTubeCenter#2131]
Since updating to 535 developer build almost every time I visit youtube the text on the whole page is flickering on/off 2 or 3 times per second. That's button labels, comments titles.. everything.
From time to time it stops flickering. 

I've tried HTML5 and Flash player still the same issue.

It's basically unusable now. What can I do to stop it?

**Comment by nomosaic** [comment]
I met same problem with you too. How to stop it

**Comment by YePpHa** [comment]
What browser are you using?

**Comment by bhrgunatha** [comment]
Sorry I forgot to say: 
Firefox 42.0. I'm on Linux (64 bit OS)
Anything else I should mention?

**Comment by YePpHa** [comment]
No, thank you. I will try to look at why this issue occurs. Hopefully, the latest update have reduced the problem.

**Comment by nomosaic** [comment]
“Flash mode” will apear this problem, but "html5 mode" is OK. I'm using firefox 42.0 on windows 7 32bit
